test/nonregression/pipelines/test_run_pipelines_dl.py

```diff
@@ -44,25 +44,32 @@ def test_run_DLPrepareData(cmdopt):
     roi_params = {
         "extract_method": "roi",
         "roi_list": ["rightHippocampusBox", "leftHippocampusBox"],
-        "roi_uncrop_output": False,
+        # "roi_uncrop_output": False,
+        "use_uncropped_image": True,
```

Comment:
I wonder what `roi_uncrop_output` is used for then?

Reply:
--use_uncropped_image is for _using_ a cropped image as input
--roi_uncrop_output is for _generating_ a cropped tensor as output